claimManagement/build.gradle

```diff
@@ -34,9 +34,21 @@ android {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+
+            buildConfigField "boolean", "LOGGING_ENABLED", "false"
```

Comment:
Is there a default value valid for all other build variants?

Reply:
@dragos-dobre Yes, for all "release" builds